### ISSUE TITLE
lack of support for external to "keystone path" development

### ISSUE THREAD

**Issue by acaldaya** [keystone-enclave/keystone#322]
For using Keystone, some env. varriables are needed.

Even if those are set, the SDK fails to locate the runtime path using this [macro](https://github.com/keystone-enclave/keystone/blob/4e966526cdcb464ad3faf8cfd1b12ca3013a2103/sdk/macros.cmake#L83)

how does it sound making it generic?
sth like:
`get_filename_component(${var} $ENV{KEYSTONE_SDK_DIR}/../../runtime REALPATH BASE_DIR "${CMAKE_SOURCE_DIR}")`

**Comment by dayeol** [comment]
Yes, it makes sense to generalize it. In the long (or mid) term, we are hoping to completely move away from the current weird build system structure. We don't really need completely separate runtime & SDK build systems after monorepo-izing everything, but we still haven't really merged them.

**Comment by acaldaya** [comment]
sure, feel free to include the snippet in your commits later.
